AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -120,6 +120,8 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 !
+ing.com#$#body { overflow: auto !important; }
+ing.com#$#.global-overlays { display: none !important; }
```

Comment:
It is used on dev domain only. The rule looks wide and dangerous.

Reply:
Can we change that like this?
```
developer.ing.com#$#body { overflow: auto !important; }
developer.ing.com##.global-overlays__backdrop
developer.ing.com##.global-overlays__overlay
```

Reply:
just change domain to developer.ing.com in original rules.

> Can we change that like this?

No. Please do not mix element hiding rules with CSS, which applied to one problem(for example, popup + scroll unblocking).

Reply:
Added directly.